### PR TITLE
Updating the Note in Viewing conditional updates in the web console

### PR DESCRIPTION
**Pull request by gagankrmahto**
Updating the Note in Viewing conditional updates in the web console document for changes.

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-36920

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by opayne1** [comment]
Hey! @gagankrmahto Great job with the PR! What versions does this apply to? I remember in slack we talked about 4.14, so should this be 4.14+?

**Review by opayne1 — COMMENTED** [review]
Just one small comment!

Link to preview: https://78826--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-cluster-web-console.html

**Comment by opayne1** [comment]
@yapei could you help with a QE review? Thank you! :)

File: modules/update-conditional-web-console.adoc [hunk 1]
```diff
@@ -27,6 +27,7 @@ You can view and assess the risks associated with particular updates with condit
 [NOTE]
 ====
 If a version with known issues is selected, more information is provided with potential risks that are associated with the version.
+If the `Include versions with known issues` feature is disabled and only Recommended versions are displayed, that means no upgrade `versions with known issues` are available.
```
[the remark this answers is not in the document]
Comment (gagankrmahto) [reply modules/update-conditional-web-console.adoc:30]: That conveys the meaning much better, thanks @opayne1

[the remark this answers is not in the document]
Comment (yapei) [reply modules/update-conditional-web-console.adoc:30]: Screenshot 2024-07-17 at 11 09 33 AM

Comment (yapei) [inline modules/update-conditional-web-console.adoc:30]:
```suggestion
If the `Include versions with known issues` feature is disabled, only recommended versions are displayed. When disabled, versions `Have known issues` are not displayed.
```

**Pull request closed without merging**